src/lib/optimizer/strategy/column_pruning_rule.cpp

```diff
@@ -74,6 +74,7 @@ ExpressionUnorderedSet ColumnPruningRule::_collect_actually_used_columns(const s
       case LQPNodeType::ShowColumns:
       case LQPNodeType::ShowTables:
       case LQPNodeType::Sort:
+      case LQPNodeType::StaticTable:
```

Comment:
@mrks happy with the name "StaticTableNode"? Does not feel overly intuitive to me, but I don't have better suggestions.

Reply:
I started typing twice and deleted it again. It is basically the LQP equivalent of the table wrapper. At the same time, StoredTableNode/GetTable have different names, too.